apps/web-login/Login.js

```diff
@@ -0,0 +1,79 @@
+// naver ouath part
+const WEB_IP = "http://192.168.0.12:8887/Login.html#";
+const CALLBACK_IP = "http://192.168.0.12:8887/CallBack.html";
+const CLIENT_ID = "Hw7RwBFOB7UjSufSEEjy";
+
+var naver_id_login = new naver_id_login(CLIENT_ID, CALLBACK_IP);
+var state = naver_id_login.getUniqState();
+naver_id_login.setButton("white", 2, 40);
+naver_id_login.setDomain(WEB_IP);
+naver_id_login.setState(state);
+naver_id_login.setPopup();
+naver_id_login.init_naver_id_login();
+
+// kakao ouath part
+Kakao.init("13cee8f0d511dff11f766fe1d031b0a1"); //발급받은 키 중 javascript키를 사용해준다.
+console.log(Kakao.isInitialized()); // sdk초기화여부판단
```

Comment:
이걸 콘솔로 출력하는 것보다 initialize가 안됐을 시 실행하지 않는쪽으로 코드를 작성하면 좋겠죠?
```suggestion
if(!Kakao.isInitialized()) {
  alert('kakao sdk is not initialized')
  return
}
```